models/nequip/filter_bad_preds.py

```
@@ -0,0 +1,23 @@
+
+# uses matbench-discovery matbench-discovery commit ID 012ccfe, k_srme commit ID 0269a946, pymatviz v0.15.1
+
+e_form_nequip_col = "e_form_per_atom_nequip"
+csv_path = "./results/nequip.csv.gz"
+if not os.path.exists(csv_path):
+    csv_path = "./nequip.csv.gz"
+
+df_preds = pd.read_csv(csv_path).set_index("material_id")
+
+# NOTE this filtering was necessary for both MACE and SevenNet because some outliers
+# have extremely low e_form (like -1e40) -- seems not to be necessary for well-trained NequIP/Allegro models
```

Ruff (0.12.2)

Line too long (108 > 88)

(E501)